src/sql/base/browser/ui/editableDropdown/browser/dropdown.ts

```diff
@@ -378,7 +380,7 @@ export class Dropdown extends Disposable implements IListVirtualDelegate<string>
 	}
 
 	private _inputValidator(value: string): IMessage | null {
-		if (!this._input.hasFocus() && this._input.isEnabled() && !this._selectList.isDOMFocused() && !this._dataSource.values.some(i => i === value)) {
```

Comment:
I removed the hasFocus part here so the error message can be displayed while the user is typing in the text box.